rules/prettier.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  rules: {
+    // http://eslint.org/docs/rules/newline-per-chained-call
+    "newline-per-chained-call": 0,
```

Comment:
We're letting prettier handle this? 😛 If I recall correctly they're using the line length as a limit. But I guess we cannot sync this.

Reply:
Unfortunately, we cannot do something for this :/

Prettier will keep the chain calls in the same line if they are less than the `printWidth`.